src/components/ProfitAndLossChart/Indicator.tsx

```diff
@@ -52,6 +48,10 @@ export const Indicator = ({
     }, 200)
   }, [animateTo])
 
+  if (!className?.match(/selected/)) {
+    return null
+  }
```

Comment:
This is one of the last `rules-of-hooks` changes. Next step is to audit dependency arrays for hook.